kimchi/build.rs

```diff
@@ -0,0 +1,11 @@
+// Build script
+
+use std::process::Command;
+
+fn main() {
+    // Rebuild specification
+    Command::new("make")
+        .args(&["-C", "../book/specifications/kimchi"])
```

Comment:
Shouldn't it be `make -C ../book/specifications kimchi build` ? Or the `build` part is intrinsic in `build.rs`?

Reply:
the first one is default according to the Makefile, but good idea to specify just in case anyone ever changes it.